distsql/distsql.go

```diff
@@ -336,10 +321,11 @@ func Analyze(ctx context.Context, client kv.Client, kvReq *kv.Request) (SelectRe
 		return nil, errors.New("client returns nil response")
 	}
 	result := &selectResult{
-		label:   "analyze",
-		resp:    resp,
-		results: make(chan resultWithErr, kvReq.Concurrency),
-		closed:  make(chan struct{}),
+		label:    "analyze",
+		resp:     resp,
+		results:  make(chan resultWithErr, kvReq.Concurrency),
+		closed:   make(chan struct{}),
+		feedback: statistics.NewQueryFeedback(0, nil, 0, false),
```

Comment:
Why add feedback here? In my opinion, we do not need feedback for Analyze statement.

Reply:
Set it here so that when updating the feedback, we do not need to check if the feedback is nil.

Reply:
But feedback is not always exists because we won't collect feedback for every sql in the final release?

Reply:
If we do not need to collect the feedback, we can set an empty one or set it to nil, both ways are ok.